conda_forge_tick/migrators.xsh

```diff
@@ -72,6 +78,47 @@ class PipMigrator(MiniMigrator):
             for b in self.bad_install:
                 replace_in_file(f'script: {b}', "script: {{ PYTHON }} -m pip install . --no-deps -vv", 'meta.yaml')
 
+
+class LicenseMigrator(MiniMigrator):
+    def filter(self, attrs: dict) -> bool:
+        license = attrs.get('meta_yaml', {}).get('about', {}).get('license')
+        license_fam = (attrs.get('meta_yaml', {}).get('about', {}).get('license_family', '').lower or
+                       license.lower.partition('-')[0].partition('v')[0])
+        if license_fam in NEEDED_FAMILIES and 'license_file' not in attrs.get('meta_yaml', {}).get('about', {}):
+            return False
+        return True
+
+    def migrate(self, recipe_dir, attrs, **kwargs):
+        # Use conda build to do all the downloading/extracting bits
+        md = MetaData(recipe_dir)
+        # go into source dir
+        cb_work_dir = provide(md)
+        with indir(cb_work_dir):
+            # look for a license file
+            license_file = next((s for s in os.listdir('.') if
+                                 any([getattr('license', k)() in s for k in ['lower', 'upper', 'title']])), None)
```

Comment:
Can you also look for `COPYING*`, `COPYRIGHT`, `LICENSE*` and if there are multiple add all of them ?